lib/custom_lib/custom_lib.h

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright (c) 2021, Legrand North America, LLC.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/* The C preprocessor namespace is global, so Zephyr regularly
+ * uses encoded pathnames in the header file multiple-inclusion guards.
+ * For example:
+ *
+ *   zephyr/lib/libc/minimal/include/sys/timespec.h uses
+ *   ZEPHYR_LIB_LIBC_MINIMAL_INCLUDE_SYS_TIMESPEC_H_
+ */
+
+#ifndef EXAMPLE_APPLICATION_LIB_CUSTOM_LIB_CUSTOM_LIB_H_
+#define EXAMPLE_APPLICATION_LIB_CUSTOM_LIB_CUSTOM_LIB_H_
+
+int custom_lib_get_value(int return_value_if_nonzero);
```

Comment:
missing docstrings

Reply:
Accepted, and manually created. Verified Zephyr docs still built.  Noticed that the CUSTOM_LIB Kconfigs are in the documentation but the added docstrings were not.  This behavior was seen elsewhere independent of this change.